wiki/Installing-drivers-and-fix-them.md

```diff
@@ -0,0 +1,78 @@
+ 2. Next, enter the ID of the device and follow the instructions on the page.
+ 3. When you selected the product, you have two options:
+	- Download the driver downloader (recommended)
+	- And download the driver on specific and run executable
+
+## Method 3: Download using the OEM's website.
+If you want to download the drivers for your GPU (Graphics Card), and the manufacturer's website doesn't provide the drivers, you can download them from the OEM's website. Only do this if you have to.
+Now follow the same instructions as Method 2, but with the OEM's website.
+
+ 1. On the search engine of your choice (Google, Bing, etc…), search for "{your brand device name} drivers".
+ 2. Search for the model number of the piece of hardware you want to upgrade/install drivers for.
+ 3. When you select the product, you have two options:
+	- It's recommended to use the driver downloader if it's provided, as it makes the installation a lot easier.
+	- Download the driver on specific and run the executable
+- See [Reinstalling GPU drivers from the OEM's website](https://msft.chat/wiki/reinstalling-gpu-drivers.html#method-3-oem-website) for more information.
+## Troubleshooting 🔧
+
+1. Enter Safe Mode:
+
+Safe mode is a diagnostic tool, which loads Windows with only the essential drivers. This allows you to access Windows even if the problematic driver is causing issues during normal boot.
+
+ - Windows 10/11: Repeatedly press and hold the power button while booting to trigger a forced restart. After 2–3 restarts, you'll see the “Advanced options” screen. Navigate to “Troubleshoot” > “Advanced options” > “Startup Settings” > “Restart now”. Upon restart, choose “Enable Safe Mode with Networking” (if you need internet) or simply “Safe Mode”.
+
+ - Earlier Windows versions: Restart your computer and repeatedly press F8 before the Windows logo appears. This should open the Advanced Boot Options menu, where you can select “Safe Mode”.
+
+2. Uninstall the Driver in Safe Mode:
+
+Once in safe mode, you can uninstall the problematic driver:
+
+ 1. Open the Device Manager (search for it in the Start Menu).
+ 2. Expand the category for the device with the recently installed driver.
+ 3. Right-click on the device and select “Uninstall device”.
+ 4. Check the box “Delete the driver software for this device” (if available) and click “Uninstall”.
+
+3. System Restore (if applicable):
+
+If you recently created a system restore point before installing the driver, you can restore Windows to that point:
+
+ 1. In Safe Mode, open the Start Menu and search for “System Restore”.
+ 2. Click “Create a restore point” and then “Next”.
+ 3. Choose the desired restore point (ideally the one before installing the driver) and click “Next” followed by “Finish”.
+
+4. Startup Repair (if Windows won't boot at all):
+
+If Safe Mode isn't accessible, you can try Startup Repair, a built-in tool on Windows that will attempt to fix boot issues:
```

Comment:
a built-in tool _**in**_ windows. (not on windows)

Reply:
![image](https://github.com/Microsoft-community/Community-website/assets/137704565/dee1bd4d-534a-4463-8d0d-b8b56d8c837a)

Reply:
Hmmm always thought that was on... Anyway ignore it for now.

Reply:
Your comment or the grammar isssue?

Reply:
> Your comment or the grammar isssue?

Both lol. You can leave this discussion open for now. But leave it as "on" in the file and commit it and we might come back to this later.

Reply:
I think 'on' would be more correct here, though I would probably just drop the "on Windows" part entirely (so "...a built-in tool") or just say something like "...a tool built-into Windows" instead